.github/workflows/cypress.yml

```
@@ -9,7 +9,10 @@ on:
     branches:
       - develop
 jobs:
+  build: 
+    uses: 10up/insert-special-characters/.github/workflows/build-release-zip.yml@update/176 # TODO: Update this to develop branch.
```

```suggestion
    uses: ./.github/workflows/build-release-zip.yml
```

This uses the file from the current branch, this may be helpful if the build needs to change alongside another PR. For example, an additional file is added.